src/lib/docoutline.ts

```diff
@@ -42,12 +39,12 @@ function build(selector) {
     })
 
     headers.forEach(item => {
-      let _el = createMenuEntry(item)
+      const _el = createMenuEntry(item)
       subMenu.appendChild(_el)
     })
   }
 
   return true
 }
 
-export default build
\ No newline at end of file
+export { build }
```

Comment:
Exporting the `build` function gives us better type information at the call site.